src/html/page_template.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 Jakub Duda
+//
+// This file is part of Poietic.
+//
+// Poietic is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License, version 2,
+// as published by the Free Software Foundation.
+//
+// Poietic is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with Poietic. If not, see <https://www.gnu.org/licenses/>.
+
+pub mod page_template_config;
+pub mod template;
+
+use std::collections::BTreeMap;
+
+use crate::html::html_safety::EscapeHtml;
+
+pub struct Meta {
+    attributes: BTreeMap<String, String>,
+}
+
+impl Meta {
+    pub fn new(attributes: BTreeMap<String, String>) -> Result<Self, ()> {
```

Comment:
This Result shouldn't use an empty tuple as its Err

Reply:
While I am working with html errors, would it be a good idea to add a string value to IllegalAttributeName? This would provide a clearer feedback to the user.

Reply:
Ok, that might be helpful